src/SegmentedArc.js

```diff
@@ -52,7 +52,7 @@ export const SegmentedArc = ({
 
   if (arcDegree > 180) {
     const offsetAngle = (arcDegree - 180) / 2;
-    const heightOffset = effectiveRadius * Math.sin(offsetAngle);
+    const heightOffset = effectiveRadius * Math.sin(offsetAngle * (Math.PI / 180));
```

Comment:
Great find! Would you be able to run `yarn test:o` and update test snapshot as there is one failing?